tools/dotnet-linker/Steps/ComputeAOTArguments.cs

```diff
@@ -52,7 +54,7 @@ protected override void TryEndProcess ()
 					item.Metadata.Add ("AOTAssembly", aotAssembly);
 					item.Metadata.Add ("LLVMFile", llvmFile);
 					item.Metadata.Add ("ObjectFile", objectFile);
-					if (isDedupAssembly.HasValue && isDedupAssembly.Value)
+					if (processingDedupAssembly)
```

Comment:
@ivanpovazan this is the build error

> Build FAILED.
>
> /Users/builder/azdo/_work/1/s/xamarin-macios/tools/dotnet-linker/Steps/ComputeAOTArguments.cs(58,45): error CS8629: Nullable value type may be null. [/Users/builder/azdo/_work/1/s/xamarin-macios/tools/dotnet-linker/dotnet-linker.csproj]
>
>    0 Warning(s)
>    1 Error(s)

Reply:
Thanks, seems I pushed something I didn't check build locally, will take a look.